core/parameters/src/view.rs

```diff
@@ -27,6 +27,7 @@ pub struct RuntimeConfigView {
     pub witness_config: WitnessConfigView,
 }
 
+/// Describes different costs for the runtime
```

Comment:
```suggestion
/// Describes different fees for the runtime
```

The distinction between two is subtle but since we use both terms in `nearcore` somewhat non-interchangeably, might as well be precise.